src/main/java/io/rainfall/execution/InParallel.java

```diff
@@ -61,7 +61,7 @@ public InParallel(final int nb, final Unit unit, final Every every, final TimeMe
   public <E extends Enum<E>> void execute(final StatisticsHolder<E> statisticsHolder, final Scenario scenario,
                                           final Map<Class<? extends Configuration>, Configuration> configurations,
                                           final List<AssertionEvaluator> assertions) throws TestException {
-    final ConcurrencyConfig concurrencyConfig = (ConcurrencyConfig)configurations.get(ConcurrencyConfig.class);
+    final ConcurrencyConfig concurrencyConfig = (ConcurrencyConfig) configurations.get(ConcurrencyConfig.class);
```

Comment:
so it seems that a number of reformatting are in that PR, maybe I missed something but usually it's better to reformat only code that was changed, or if reformatting is important for some specific purpose, make it part of a specific PR